docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -38,6 +38,12 @@ components.gateway.apiml.service.additionalRegistration:
                 serviceUrl: /
 ```
 
+```
+components.gateway.apiml.security.x509:
+    #  cloud gateway port 
+        certificatesUrl: https://ca32.lvn.broadcom.net:27558/gateway/certificates
```

Comment:
Please, remove real host, port and system names from everywhere.

Reply:
I think this branch needs to be rebased with docs-staging as there are conflicts.